src/kirby/components/list/list-item-color.directive.ts

```diff
@@ -0,0 +1,22 @@
+import { Directive, Input, Renderer2, ElementRef, OnChanges, SimpleChanges } from '@angular/core';
+
+@Directive({
+  selector: '[kirbyListItemColor]',
+})
+export class ListItemColorDirective implements OnChanges {
+  @Input()
+  colorCb: (item: any) => string;
```

Comment:
Lets talk about naming 😄 

We have no where in codebase used this naming scheme. I think most (if not all) places we name like `getSectionName`. I don't have any strong feeling on the name it self. But i feel like we should align the naming.

This goes for all the places you use `colorCb`.